src/servers/graphql-server.ts

```diff
@@ -42,10 +43,22 @@ export const startApolloServer = async ({
   startSubscriptionServer = false,
 }): Promise<Record<string, unknown>> => {
   const app = express()
+
+  const apolloPulgins = isProd
```

Comment:
did you test this with credentials in your local environment? if this fail we will only find out when it is deployed to production

Reply:
Should definitely verify its working with real creds locally once before deploy - I agree. Has this been checked @samerbuna ?

Reply:
Yep. However, errors thrown in the context don't seem to trigger it 😞

Reply:
That sucks... guess we can merge this anyhow as it provides value, but still need to find another solution for the `context` errors.